packages/origin-247-certificate/src/offchain-certificate/synchronize/strategies/batch/batch.configuration.ts

```diff
@@ -6,8 +9,19 @@ export interface BatchConfiguration {
 
 export const BATCH_CONFIGURATION_TOKEN = Symbol.for('BATCH_CONFIGURATION_TOKEN');
 
-export const batchConfiguration: BatchConfiguration = {
-    issueBatchSize: 10,
-    claimBatchSize: 25 /** @NOTE this may be further optimized probably */,
-    transferBatchSize: 100
-};
+@Injectable()
+export class BatchConfigurationService {
+    constructor(private configService: ConfigService) {}
+
+    getBatchSizes(): BatchConfiguration {
+        const issueSize = this.configService.get('ISSUE_BATCH_SIZE');
+        const claimSize = this.configService.get('CLAIM_BATCH_SIZE');
+        const transferSize = this.configService.get('TRANSFER_BATCH_SIZE');
+
+        return {
+            issueBatchSize: issueSize ? parseInt(issueSize) : 10,
```

Comment:
default values are in two places, maybe it would be better to have it all managed by configuration service and make it non-nullable non-undefineable value